ee/codegen/src/__test__/__snapshots__/workflow.test.ts.snap

```diff
@@ -201,3 +201,14 @@ class TestWorkflow(BaseWorkflow[Inputs, BaseState]):
     graph = SearchNode
 "
 `;
+
+exports[`Workflow > write > should handle the case of multiple nodes with the same label 1`] = `
+"from vellum.workflows import BaseWorkflow
+from .nodes.templating_node import TemplatingNode
+from .nodes.templating_node_1 import TemplatingNode as TemplatingNode1TemplatingNode
+
+
+class TestWorkflow(BaseWorkflow):
+    graph = TemplatingNode >> TemplatingNode
```

Comment:
We aren't yet using the correct aliased class names here. This requires a fern-side fix, which is up for review here: https://github.com/fern-api/fern/pull/5401